config/environments/development.rb

```diff
@@ -1,4 +1,6 @@
 require "active_support/core_ext/integer/time"
+require 'factory_bot_rails'
```

Comment:
will your seeds work on Heroku if you require this only for development and test envs?

Reply:
I think no, but I can push my db on Heroku `heroku pg:push ...` and we will have our data there . Should I redo this for a production environment?

Reply:
yes. you can just require it in application.rb
it's not a good idea to push your db to heroku. it's better to have an empty db there and run seeds in heroku console

Reply:
done